sdk/java-sdk-spring/src/main/scala/kalix/spring/impl/KalixSpringApplication.scala

```diff
@@ -249,6 +262,7 @@ case class KalixSpringApplication(applicationContext: ApplicationContext, config
 
   private val messageCodec = new JsonMessageCodec
   private val kalixClient = new RestKalixClientImpl(messageCodec)
+  private val componentClient = new ComponentClient(kalixClient)
```

Comment:
We could make `KalixClient` and `ComponentClient` public and remove the getters. They are immutable and that's also internal code